docs/download/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: Download
+hide:
+  #- navigation
+  - toc
+---
+
+By installing you agree to the [Omnissa general terms](https://www.omnissa.com/general-terms/)
+
+
+| Horizon Server version                                                                                                         | Download Powercli module |
+|------------------------------------------------------------------------------------------------------------------------| --- |
+| 8.13/2406                    | [horizon4powercli_2406](https://artifactory.air-watch.com:443/artifactory/github-publish-local/euc-vdi/horizonpowercli/build-horizon4powercli/103/build/1/horizon4powercli.zip) |
```

Comment:
I don't believe this URL is accessible from outside of the corporate network.

Please update with a externally accessible URL.

Reply:
Quick confirmation on my cell phone browser shows "Server can't be found" for http://artifactory.air-watch.com/

Reply:
fixed